### ISSUE TITLE
Allow swipe in intro screen

### ISSUE THREAD

**Issue by NicoWeio** [syncthing/syncthing-android#1334]
I think the title says it all. I find it to be absolutely common, and I was somewhat irritated when swiping on the intro screen wouldn't do anything.
As far as I know (I once did it myself), this is quite straightforward to implement.

**Comment by Catfriend1** [comment]
Yes that would be nice. I just was unable to find the right working implementation when I created the slides. One problem though: Swipe shouldn't be the only way because it would make the app no longer confirm the dpad controller requirements for Google play tv app eligibility.

**Comment by Lythenas** [comment]
Not sure if this was implemented by now. Maybe this should be it's own issue.

I found out that if I swipe from right to left it tries to move to the next screen but it does not quite work. The screen is only moved by a small random amount (as little as 1/10 of the screen). And I can continue to move it by swiping. But even if I reach the next screen completely it does not update the dots at the bottom of the screen.
